packages/core/contracts/financial-templates/optimistic-distributor/OptimisticDistributor.sol

```diff
@@ -181,13 +174,14 @@ contract OptimisticDistributor is Lockable, MultiCaller, Testable {
         // Store funded reward and log created reward.
         Reward memory reward =
             Reward({
-                distributionProposed: DistributionProposed.None,
+                distributionExecuted: false,
                 sponsor: msg.sender,
                 rewardToken: rewardToken,
                 maximumRewardAmount: maximumRewardAmount,
                 earliestProposalTimestamp: earliestProposalTimestamp,
                 optimisticOracleProposerBond: optimisticOracleProposerBond,
                 optimisticOracleLivenessTime: optimisticOracleLivenessTime,
+                blockingProposalTimestamp: 0,
```

Comment:
I dont see where this variable is ever set to anything other than 0. you do `if (reward.blockingProposalTimestamp == 0) return true;` but where do you set it to a non-0 value?

Reply:
nvm it's https://github.com/UMAprotocol/protocol/blob/99cdc5448748ef8a13f99a6f98d5f698e5b48751/packages/core/contracts/financial-templates/optimistic-distributor/OptimisticDistributor.sol#L248